src/docs/Heatmap.md

```diff
@@ -0,0 +1,17 @@
+# Heatmap design docs
```

Comment:
Well done doing this without the designs! [But they are actually there](https://app.zeplin.io/project/5e40457df723790a43273103/screen/5e4cfbc201c6a2a9a94587a3) and not just an image. Sorry for the additional work. In such a case you can ask the designer (in this case me) if there has been a mistake. Everything should be a proper design.

![image](https://user-images.githubusercontent.com/9722792/78536717-93929380-77ee-11ea-8309-ecbe27b1c670.png)